File: frontend/lib/features/accounting/domain/entities/accounting_voucher_line.dart
class AccountingVoucherLine {
  const AccountingVoucherLine({
    required this.id,
    required this.voucherId,
    required this.lineNo,
    required this.accountId,
    required this.accountCode,
    required this.accountName,
    required this.summary,
    required this.debitAmount,
    required this.creditAmount,
  });

  final String id;
  final String voucherId;
  final int lineNo;
  final String accountId;
  final String accountCode;
  final String accountName;
  final String summary;
  final double debitAmount;
  final double creditAmount;
}

File: backend/src/main/java/com/microflow/accounting/domain/model/AccountingVoucherLine.java
package com.microflow.accounting.domain.model;

import java.math.BigDecimal;

public record AccountingVoucherLine(
        String id,
        String voucherId,
        int lineNo,
        String accountId,
        String accountCode,
        String accountName,
        String summary,
        BigDecimal debitAmount,
        BigDecimal creditAmount
) {
}
